components/eamxx/cime_config/namelist_defaults_scream.xml

```diff
@@ -231,6 +231,8 @@ be lost if SCREAM_HACK_XML is not enabled.
       <nudging_filename type="array(string)"/>
       <nudging_fields type="array(string)" doc="List of fields to be nudged.  Note, syntax of 'A:B' represents nudging field A with data from field B in files, syntax of 'A' assumes that nudging file has the same variables name as EAMxx"/>
       <nudging_timescale type="integer" doc="Timescale to apply nudging tendencies, 0: full replacement, >0: actual timescale">0</nudging_timescale>
+      <use_nudging_weights type="integer" doc="Flag for nudging weights option"/>
```

Comment:
Would it make more sense to have "use_nudging_weights" be true or false rather than an integer? Is zero false and 1 true?

Reply:
yeah, zero is false, and 1 is true, 
xml doesn't support _bool_, and _boolean_, see the error information below:
NameError: ERROR: Invalid/unsupported force type 'boolean', and 
NameError: ERROR: Invalid/unsupported force type 'bool'

Reply:
Ah, we are using `logical` in our XML settings. Can you try with that?